### DISCUSSION TITLE
[Chandler] Is it possible to disable the registration?

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [monicahq/monica#6986] [Q&A]
Heho
Is it possible to disable the user registration at the moment?
I tried APP_DISABLE_SIGNUP but this does not work and I searched for it and did't found anything in the code.
Isn't it implemented yet?

**Comment by Seantourage** [comment]
Noticing the same issue here

**Reply by quentinbonge95** [reply]
Best workaround is to enable MAIL_MAILER=log and click on the link in the logged email once you've "signed up"

**Reply by JCBerger** [reply]
How do you receive the email, though?

**Comment by JCBerger** [comment]
I'm wondering if I'm dealing with the same problem. I've tried to create a new account, but the system then tries to send a verification email. Since I'm running this locally through Docker, it doesn't send one. I've tried to find where I could add this text, or a way to modify the database and add a user account in there, but I'm stumped. Could someone help me out?

**Comment by davpsh** [comment]
Issue fixed in [v5.0.0-beta.5](https://github.com/monicahq/monica/releases/tag/v5.0.0-beta.5) and you can use `APP_DISABLE_SIGNUP` env setting now.
